### DISCUSSION TITLE
IEA-15-MW-RWT not generating any power

### DISCUSSION THREAD

**Discussion by clbanjai** [OpenFAST/openfast#2344] [Q&A] [answered]
Hello,

I have been trying to simulate the the 15MW turbine on OpenFAST but for some reason, I can't get it to produce any power. I see that the GenSpeed has an average of 7 rpm over the course of 200 seconds of simulation time but for some reason GenPwr and GenTq do not budge from 0. 

Thanks for the help.

**Comment by jjonkman** [comment, marked as answer]
Dear @clbanjai,

Are you using the baseline ROSCO controller for the IEA Wind 15-MW RWT by setting `PCMode` = `VSContrl` = 5 in ServoDyn, with `DLL_FileName` locating the appropriate dynamic library?  Do you see ROSCO being called in the OpenFAST messages written to the command prompt when you execute the simulation?

Best regards,

**Reply by clbanjai** [reply]
The line with DLL_FileName was pointing towards the discon.dll when it should have been pointing towards libdison.dll. Now I have my turbine generating power, thank you very much!
